orchestration/dagster_orchestration/hca_orchestration/tests/solids/load_hca/test_load_table.py

```diff
@@ -1,120 +1,100 @@
 from unittest.mock import Mock
 
-from dagster import ModeDefinition, ResourceDefinition, SolidExecutionResult, execute_solid, resource
-from hca_orchestration.contrib.bigquery import BigQueryService
+import pytest
+from dagster import resource, ModeDefinition, ResourceDefinition, SolidExecutionResult, execute_solid
 from dagster_utils.contrib.data_repo.typing import JobId
+from google.cloud.storage import Client, Blob
 
+from hca_orchestration.contrib.bigquery import BigQueryService
+from hca_orchestration.contrib.data_repo import DataRepoService
+from hca_orchestration.models.hca_dataset import TdrDataset
+from hca_orchestration.models.scratch import ScratchConfig
 from hca_orchestration.pipelines.load_hca import test_mode
-from hca_orchestration.solids.load_hca.load_table import check_has_data, check_has_outdated, clear_outdated, \
-    load_table, start_load
+from hca_orchestration.solids.load_hca.load_table import load_table_solid, clear_outdated
 from hca_orchestration.support.typing import HcaScratchDatasetName, MetadataType, MetadataTypeFanoutResult
 from hca_orchestration.tests.support.gcs import FakeGCSClient, FakeGoogleBucket, HexBlobInfo
 
-test_bucket = FakeGoogleBucket(
-    {"gs://my-fake-bucket/fake-prefix": HexBlobInfo(hex_md5="b2d6ec45472467c836f253bd170182c7", content="test content")}
-)
 
-test_bucket_name = "my-fake-bucket"
-
-load_table_test_mode = ModeDefinition(
-    "test_load_table",
-    resource_defs={**test_mode.resource_defs}
-)
-load_table_test_mode.resource_defs["gcs"] = ResourceDefinition.hardcoded_resource(
-    FakeGCSClient(
-        buckets={test_bucket_name: test_bucket}
-    )
-)
-
-run_config = {
```

Comment:
Rather than having bare hardcoded constants, I've wrapped all of these in `pytest.fixture` fixture functions below.